packages/medusa/src/api/routes/admin/workflows-executions/get-execution.ts

```diff
@@ -0,0 +1,26 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IWorkflowEngineService } from "@medusajs/workflows-sdk"
+import { MedusaRequest, MedusaResponse } from "../../../../types/routing"
+
+export default async (req: MedusaRequest, res: MedusaResponse) => {
+  const workflowEngineService: IWorkflowEngineService = req.scope.resolve(
+    ModuleRegistrationName.WORKFLOW_ENGINE
+  )
+
+  const { id, workflow_id, transaction_id } = req.params
+
+  const execution = await workflowEngineService.retrieveWorkflowExecution(
+    id ?? {
+      workflow_id,
+      transaction_id,
```

Comment:
See comment on URLs. We could use a standard filter approach for this query

Reply:
not sure I get it. What is the standard filter for the retrieve?
here the primary key is workflow_id + transaction_id, but we also have an id.

Reply:
My thinking was that we could just use a regular list endpoint when you want to query using the two IDs:
```
GET /admin/workflow-executions?workflow_id=,transaction_id,
```
Instead of having this multi-purpose endpoint.

But I am fine to keep it as is 👍

Reply:
for a retrieve where just one single entity is expected, I believe all our endpoints are using just the url, no querystring.
But I'm not aware of other endpoint that could be `/something/_id` or `/something/pk_1/pk_2`